core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -892,7 +892,7 @@ protected function generateGlobalButtons()
 
 			$v = \is_array($v) ? $v : array($v);
 			$label = \is_array($v['label']) ? $v['label'][0] : $v['label'];
-			$title = \is_array($v['label']) ? $v['label'][1] : $v['label'];
+			$title = \is_array($v['label']) ? ($v['label'][1] ?? '') : $v['label'];
```

Comment:
```suggestion
			$title = \is_array($v['label']) ? ($v['label'][1] ?? null) : $v['label'];
```

Otherwise it is a behavior change.